sdks/python/src/opik/api_objects/helpers.py

```diff
@@ -1,11 +1,13 @@
 import datetime
 import logging
-from typing import Optional, Dict, Any
+from typing import Optional, Dict, Any, List
 
 from opik import llm_usage
 
 from .. import config, datetime_helpers, logging_messages
 from ..id_helpers import generate_id  # noqa: F401 , keep it here for backward compatibility with external dependants
+from ..rest_api import SpanFilterPublic
```

Comment:
Please import modules. If the module name itself is not very descriptive, use "as" to add more context.
For example
`from ..rest_api import core as rest_api_core`